### DISCUSSION TITLE
Splunk to vector

### DISCUSSION THREAD

**Discussion by gitingua** [vectordotdev/vector#16082] [General]
Colleagues hello everyone. 
I have this question.
did someone set up sending with Splunk Universal Forwarders to kafka ? If yes. please tell me

**Comment by davidhuie-dd** [comment]
Hi @gitingua, this should be possible by configuring a Universal Forwarder to emit HEC logs to Vector. You can then forward the logs to Kafka via Vector. There is some documentation for configuring HEC forwarding in the Universal Forwarder here: https://docs.splunk.com/Documentation/Forwarder/9.0.2/Forwarder/Configureforwardingwithoutputs.conf
